js/main.js

```diff
@@ -0,0 +1,16 @@
+const getRandomInt = (firstValue, secondValue) => {
+  firstValue = Math.ceil(firstValue);
+  secondValue = Math.floor(secondValue);
+  if (firstValue < 0 || secondValue < 0 || (secondValue - firstValue) < 0) {return null;}
```

Comment:
```suggestion
  if (firstValue < 0 || secondValue < 0 || (secondValue - firstValue) < 0) {
    return null;
  }
```
В JS принять писать `if` не в одну строку, по этому сразу пиши красиво 🙂